src/app/content/components/Toolbar/HighlightButton.tsx

```diff
@@ -22,6 +22,12 @@ const MyHighlightsWrapper = styled(PlainButton)`
   align-items: center;
   justify-content: center;
   margin-right: 2rem;
+  height: auto;
+
+  :focus {
+    outline: auto -webkit-focus-ring-color;
```

Comment:
what is this for?

Reply:
if I dont put this, the outline is gray on focus, I assumed blue outline is expected.

Reply:
i would leave it as default, i see normal behavior is grey outline when focused and blue when active, but i'm not sure, we should just leave whatever the browser does by default since we have no custom states